### PR TITLE
add showBitcoinWallet to portis btcGetAddress

### PR DESCRIPTION
**Pull request by natven**

### CODE REVIEW

File: packages/hdwallet-portis/src/portis.ts [hunk 1]
```diff
@@ -234,6 +234,10 @@ export class PortisHDWallet implements HDWallet, ETHWallet, BTCWallet {
         throw new Error(`Unsupported scriptType ${scriptType}`)
     }
 
+    if(msg.showDisplay === true) {
+      this.portis.showBitcoinWallet(b32string)
```
[the remark this answers is not in the document]
Comment (natven) [reply packages/hdwallet-portis/src/portis.ts]: couldn't we get into that situation regardless? the point of this is to verify we're on the same page with the device (portis in this case) and showing the user that nothing malicious is happening with address injection. we're counting on them (portis) to interpret the path we give them correctly.

I don't see a lot of value in verifying our ability to pass 2 arguments correctly vs 1. maybe I'm missing the point.

[the remark this answers is not in the document]
Comment (natven) [reply packages/hdwallet-portis/src/portis.ts]: ok.

so adding a check to btcGetAddress that verifies:
addressNList[0] === 0x80000000 + 44 && scriptType = 'p2pkh'
OR
addressNList[0] === 0x80000000 + 49 && scriptType = 'p2sh-p2wpkh'
OR
addressNList[0] === 0x80000000 + 84 && scriptType = 'p2wpkh'

and throws when false satisfies your concerns?

[the remark this answers is not in the document]
Comment (keepkeyjon) [reply packages/hdwallet-portis/src/portis.ts]: Yeah. Only matters in the `showDisplay === true` case.

[the remark this answers is not in the document]
Comment (natven) [reply packages/hdwallet-portis/src/portis.ts]: 👍

**Pull request merged**